pkg/cmd/builder.go

```diff
@@ -110,7 +114,9 @@ func (b *AdapterBase) InstallFlags() {
 			"kubeconfig file pointing at the 'core' kubernetes server with enough rights to list "+
 				"any described objects")
 		b.FlagSet.DurationVar(&b.DiscoveryInterval, "discovery-interval", b.DiscoveryInterval,
-			"interval at which to refresh API discovery information")
+			"Interval at which to refresh API discovery information")
+		b.FlagSet.Float32Var(&b.ClientQPS, "client-qps", rest.DefaultQPS, "Maximum QPS for client")
```

Comment:
we should make the description of this command consistent with the one for burst:
```
Maximum QPS for client-side throttle
```

Reply:
already edited